src/components/UnseenBadge/UnseenBadge.tsx

```diff
@@ -3,15 +3,37 @@ import { useKnockFeed } from "../KnockFeedProvider";
 import { formatBadgeCount } from "../../utils";
 
 import "./styles.css";
+import { FeedMetadata } from "@knocklabs/client";
 
-export const UnseenBadge = () => {
+export type BadgeCountType = "unseen" | "unread" | "all";
+
+export type UnseenBadgeProps = {
+  badgeCountType?: BadgeCountType;
+};
+
+function selectBadgeCount(badgeCount: BadgeCountType, metadata: FeedMetadata) {
```

Comment:
This variable name `badgeCount` feels rather unintuitive to me, I think we meant to say `badgeCountType`?

Reply:
Fair! Will fix